pkgs/top-level/all-packages.nix

```diff
@@ -29266,7 +29266,10 @@ with pkgs;
 
   keepassx = callPackage ../applications/misc/keepassx { };
   keepassx2 = callPackage ../applications/misc/keepassx/2.0.nix { };
-  keepassxc = libsForQt5.callPackage ../applications/misc/keepassx/community.nix { };
+  keepassxc = libsForQt5.callPackage ../applications/misc/keepassx/community.nix {
+    inherit (darwin.apple_sdk_11_0.frameworks) LocalAuthentication;
+    stdenv = if stdenv.isDarwin then darwin.apple_sdk_11_0.stdenv else stdenv;
+  };
```

Comment:
Is there an advantage to doing this here instead of in `pkgs/applications/misc/keepassx/community.nix`?

Reply:
@turion , I thought about that too. The only advantage I could see to keeping it here is the consistency with other packages.

I saw the same pattern with 37 other packages, in `all-packages.nix`, by searching for `apple_sdk_11_0.frameworks`.

Reply:
Maybe there is a difference between overriding `libsForQt5.callPackage` and `callPackage`, but if there is I don't know it.

Ok, then let's keep it consistent and hope that whoever will update the SDK eventually will also remember to search for that code :sweat_smile: